backend/api/rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "stable"
```

Comment:
@RickyLB the backend builds in CI with stable afaik, I've set this here so that I don't need to switch toolchains manually.